### PR TITLE
Refine BZ issue leading keywords.

### PR DESCRIPTION
**Pull request by imtayadeway**
Adds case-insensitive varations of 'close', 'fix' and 'resolve'.

Closes https://github.com/ManageIQ/miq_bot/issues/72

### CODE REVIEW

File: lib/miq_tools_services/bugzilla.rb [hunk 1]
```diff
@@ -26,7 +40,7 @@ def service
     def self.ids_in_git_commit_message(message)
       ids = []
       message.each_line.collect do |line|
-        match = %r{^(Fixes|Resolves)?\s*https://bugzilla\.redhat\.com//?show_bug\.cgi\?id=(?<bug_id>\d+)$}.match(line)
+        match = /^(#{CLOSING_KEYWORDS.join("|")})?\s*#{URL_REGEX}$/i.match(line)
```
Comment (blomquisg) [inline lib/miq_tools_services/bugzilla.rb]: Maybe also allow colon (":") here.  People are sometimes gonna type:

> Fixes: &lt;url&gt;

We should handle both cases (with and without).

Comment (imtayadeway) [inline lib/miq_tools_services/bugzilla.rb]: done!

File: lib/miq_tools_services/bugzilla.rb [hunk 2]
```diff
@@ -26,7 +40,7 @@ def service
     def self.ids_in_git_commit_message(message)
       ids = []
       message.each_line.collect do |line|
-        match = %r{^(Fixes|Resolves)?\s*https://bugzilla\.redhat\.com//?show_bug\.cgi\?id=(?<bug_id>\d+)$}.match(line)
+        match = /^((#{CLOSING_KEYWORDS.join("|")}):?)?\s*#{URL_REGEX}$/i.match(line)
```
Comment (chessbyte) [inline lib/miq_tools_services/bugzilla.rb]: should we extract the creation of the RegEx into a constant on the same level as `URL_REGEX`?

`MATCH_REGEX = /^((#{CLOSING_KEYWORDS.join("|")}):?)?\s*#{URL_REGEX}$/i`

then this line becomes:

` match = MATCH_REGEX.match(line)`

Comment (imtayadeway) [inline lib/miq_tools_services/bugzilla.rb]: :thumbsup: i wanted to do this but couldn't think of a good name. thanks!

**Comment by blomquisg** [comment]
:+1: (I don't have the magic merge button in this repo)

**Pull request merged**